2012/05/12 16:40:56 [error] [php] PDO::__construct() [<a href='pdo.--construct'>pdo.--construct</a>]: [2002] No such file or directory (trying to connect via unix:///var/mysql/mysql.sock) (/Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php:405)
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(313): CDbConnection->open()
#1 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(291): CDbConnection->setActive()
#2 /Users/xiaorich/www/atzhuhai/framework/base/CModule.php(372): CDbConnection->init()
#3 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(429): CWebApplication->getComponent()
#4 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(609): CWebApplication->getDb()
#5 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(2275): Event->getDbConnection()
#6 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(372): CActiveRecordMetaData->__construct()
#7 /Users/xiaorich/www/atzhuhai/protected/models/Event.php(25): model()
#8 /Users/xiaorich/www/atzhuhai/protected/controllers/HomeController.php(43): model()
#9 /Users/xiaorich/www/atzhuhai/framework/web/actions/CInlineAction.php(50): HomeController->actionIndex()
#10 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(300): CInlineAction->runWithParams()
#11 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(134): HomeController->runAction()
#12 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilter.php(41): CFilterChain->run()
#13 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(1144): CAccessControlFilter->filter()
#14 /Users/xiaorich/www/atzhuhai/framework/web/filters/CInlineFilter.php(59): HomeController->filterAccessControl()
#15 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter()
#16 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(283): CFilterChain->run()
#17 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(257): HomeController->runActionWithFilters()
#18 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(277): HomeController->run()
#19 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController()
#20 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#21 /Users/xiaorich/www/atzhuhai/index-test.php(15): CWebApplication->run()
REQUEST_URI=/index-test.php
2012/05/12 16:40:57 [error] [php] PDO::__construct() [<a href='pdo.--construct'>pdo.--construct</a>]: [2002] No such file or directory (trying to connect via unix:///var/mysql/mysql.sock) (/Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php:405)
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(313): CDbConnection->open()
#1 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(291): CDbConnection->setActive()
#2 /Users/xiaorich/www/atzhuhai/framework/base/CModule.php(372): CDbConnection->init()
#3 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(429): CWebApplication->getComponent()
#4 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(609): CWebApplication->getDb()
#5 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(2275): Event->getDbConnection()
#6 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(372): CActiveRecordMetaData->__construct()
#7 /Users/xiaorich/www/atzhuhai/protected/models/Event.php(25): model()
#8 /Users/xiaorich/www/atzhuhai/protected/controllers/HomeController.php(43): model()
#9 /Users/xiaorich/www/atzhuhai/framework/web/actions/CInlineAction.php(50): HomeController->actionIndex()
#10 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(300): CInlineAction->runWithParams()
#11 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(134): HomeController->runAction()
#12 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilter.php(41): CFilterChain->run()
#13 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(1144): CAccessControlFilter->filter()
#14 /Users/xiaorich/www/atzhuhai/framework/web/filters/CInlineFilter.php(59): HomeController->filterAccessControl()
#15 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter()
#16 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(283): CFilterChain->run()
#17 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(257): HomeController->runActionWithFilters()
#18 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(277): HomeController->run()
#19 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController()
#20 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#21 /Users/xiaorich/www/atzhuhai/index-test.php(15): CWebApplication->run()
REQUEST_URI=/index-test.php
2012/05/12 16:41:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'u116143'@'localhost' (using password: YES)' in /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php:364
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(313): CDbConnection->open()
#1 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(291): CDbConnection->setActive(true)
#2 /Users/xiaorich/www/atzhuhai/framework/base/CModule.php(372): CDbConnection->init()
#3 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(429): CModule->getComponent('db')
#4 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(609): CApplication->getDb()
#5 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(2275): CActiveRecord->getDbConnection()
#6 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(372): CActiveRecordMetaData->__construct(Object(Event))
#7 /Users/xiaorich/www/atzhuhai/protected/models/Event.php(25): CActiveRecord::model('Event')
#8 /Users/xiaorich/www/atzhuhai/protected/controllers/HomeController.php(43): Event::model()
#9 /Users/xiaorich/www/atzhuhai/framework/web/actions/CInlineAction.php(50): HomeController->actionIndex()
#10 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(300): CInlineAction->runWithParams(Array)
#11 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#12 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilter.php(41): CFilterChain->run()
#13 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(1144): CFilter->filter(Object(CFilterChain))
#14 /Users/xiaorich/www/atzhuhai/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#15 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#16 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(283): CFilterChain->run()
#17 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(277): CController->run('')
#19 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('')
#20 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#21 /Users/xiaorich/www/atzhuhai/index-test.php(15): CApplication->run()
#22 {main} REQUEST_URI=/index-test.php
2012/05/12 16:41:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'u116143'@'localhost' (using password: YES)' in /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php:364
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(313): CDbConnection->open()
#1 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(291): CDbConnection->setActive(true)
#2 /Users/xiaorich/www/atzhuhai/framework/base/CModule.php(372): CDbConnection->init()
#3 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(429): CModule->getComponent('db')
#4 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(609): CApplication->getDb()
#5 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(2275): CActiveRecord->getDbConnection()
#6 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(372): CActiveRecordMetaData->__construct(Object(Event))
#7 /Users/xiaorich/www/atzhuhai/protected/models/Event.php(25): CActiveRecord::model('Event')
#8 /Users/xiaorich/www/atzhuhai/protected/controllers/HomeController.php(43): Event::model()
#9 /Users/xiaorich/www/atzhuhai/framework/web/actions/CInlineAction.php(50): HomeController->actionIndex()
#10 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(300): CInlineAction->runWithParams(Array)
#11 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#12 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilter.php(41): CFilterChain->run()
#13 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(1144): CFilter->filter(Object(CFilterChain))
#14 /Users/xiaorich/www/atzhuhai/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#15 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#16 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(283): CFilterChain->run()
#17 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(277): CController->run('')
#19 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('')
#20 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#21 /Users/xiaorich/www/atzhuhai/index-test.php(15): CApplication->run()
#22 {main} REQUEST_URI=/index-test.php
2012/05/12 16:41:55 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'u116143'@'localhost' (using password: YES)' in /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php:364
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(313): CDbConnection->open()
#1 /Users/xiaorich/www/atzhuhai/framework/db/CDbConnection.php(291): CDbConnection->setActive(true)
#2 /Users/xiaorich/www/atzhuhai/framework/base/CModule.php(372): CDbConnection->init()
#3 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(429): CModule->getComponent('db')
#4 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(609): CApplication->getDb()
#5 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(2275): CActiveRecord->getDbConnection()
#6 /Users/xiaorich/www/atzhuhai/framework/db/ar/CActiveRecord.php(372): CActiveRecordMetaData->__construct(Object(Event))
#7 /Users/xiaorich/www/atzhuhai/protected/models/Event.php(25): CActiveRecord::model('Event')
#8 /Users/xiaorich/www/atzhuhai/protected/controllers/HomeController.php(43): Event::model()
#9 /Users/xiaorich/www/atzhuhai/framework/web/actions/CInlineAction.php(50): HomeController->actionIndex()
#10 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(300): CInlineAction->runWithParams(Array)
#11 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#12 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilter.php(41): CFilterChain->run()
#13 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(1144): CFilter->filter(Object(CFilterChain))
#14 /Users/xiaorich/www/atzhuhai/framework/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#15 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#16 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(283): CFilterChain->run()
#17 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(277): CController->run('')
#19 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('')
#20 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#21 /Users/xiaorich/www/atzhuhai/index-test.php(15): CApplication->run()
#22 {main} REQUEST_URI=/index-test.php
2012/05/12 16:42:13 [error] [php] strtotime() [<a href='function.strtotime'>function.strtotime</a>]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'Asia/Chongqing' for 'CST/8.0/no DST' instead (/Users/xiaorich/www/atzhuhai/protected/views/home/index.php:42)
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/web/CBaseController.php(88): HomeController->renderInternal()
#1 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(866): HomeController->renderFile()
#2 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(779): HomeController->renderPartial()
#3 /Users/xiaorich/www/atzhuhai/protected/controllers/HomeController.php(73): HomeController->render()
#4 /Users/xiaorich/www/atzhuhai/framework/web/actions/CInlineAction.php(50): HomeController->actionIndex()
#5 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(300): CInlineAction->runWithParams()
#6 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(134): HomeController->runAction()
#7 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilter.php(41): CFilterChain->run()
#8 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(1144): CAccessControlFilter->filter()
#9 /Users/xiaorich/www/atzhuhai/framework/web/filters/CInlineFilter.php(59): HomeController->filterAccessControl()
#10 /Users/xiaorich/www/atzhuhai/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter()
#11 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(283): CFilterChain->run()
#12 /Users/xiaorich/www/atzhuhai/framework/web/CController.php(257): HomeController->runActionWithFilters()
#13 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(277): HomeController->run()
#14 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController()
#15 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#16 /Users/xiaorich/www/atzhuhai/index-test.php(15): CWebApplication->run()
REQUEST_URI=/index-test.php
2012/05/12 16:46:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "web/css/city_block_menu.gif".' in /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php:281
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('web/css/city_bl...')
#1 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /Users/xiaorich/www/atzhuhai/index.php(14): CApplication->run()
#3 {main} REQUEST_URI=/web/css/city_block_menu.gif
2012/05/12 16:46:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "web/css/city_block_menu.gif".' in /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php:281
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('web/css/city_bl...')
#1 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /Users/xiaorich/www/atzhuhai/index.php(14): CApplication->run()
#3 {main} REQUEST_URI=/web/css/city_block_menu.gif
2012/05/26 17:54:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php:281
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('favicon.ico')
#1 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /Users/xiaorich/www/atzhuhai/index.php(14): CApplication->run()
#3 {main} REQUEST_URI=/favicon.ico
2012/05/26 17:54:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php:281
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('favicon.ico')
#1 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /Users/xiaorich/www/atzhuhai/index.php(14): CApplication->run()
#3 {main} REQUEST_URI=/favicon.ico
2012/05/28 18:30:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php:281
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('favicon.ico')
#1 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /Users/xiaorich/www/atzhuhai/index.php(14): CApplication->run()
#3 {main} REQUEST_URI=/favicon.ico
2012/05/28 18:30:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php:281
Stack trace:
#0 /Users/xiaorich/www/atzhuhai/framework/web/CWebApplication.php(136): CWebApplication->runController('favicon.ico')
#1 /Users/xiaorich/www/atzhuhai/framework/base/CApplication.php(158): CWebApplication->processRequest()
#2 /Users/xiaorich/www/atzhuhai/index.php(14): CApplication->run()
#3 {main} REQUEST_URI=/favicon.ico
